### PR TITLE
Fixed the issues related to the GDI-DE compliance.

### PR DESCRIPTION
**Pull request by ahmdthr**
The current generated ISO metadata conforms to the ISO-19139 standards, but fails the GDI-DE compliance. This PR aims to fix the GDI-DE compliance issues.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/GeoNode/geonode/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in the documentation)
- [ ] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for contrib modules):
- [ ] There is a ticket in https://github.com/GeoNode/geonode/issues describing the issue/improvement/feature (a notable exemption is, changes not visible to end-users)
- [ ] The issue connected to the PR must have Labels and Milestone assigned
- [ ] PR for bug fixes and small new features are presented as a single commit
- [ ] Commit message must be in the form "[Fixes #<issue_number>] Title of the Issue"
- [ ] New unit tests have been added covering the changes, unless there is an explanation on why the tests are not necessary/implemented
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] This PR passes the QA checks: black geonode && flake8 geonode
- [ ] Commits changing the **settings**, **UI**, **existing user workflows**, or adding **new functionality**, need to include documentation updates
- [ ] Commits adding **new texts** do use gettext and have updated .po / .mo files (without location infos)

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.**

### CODE REVIEW

File: geonode/catalogue/templates/catalogue/full_metadata.xml [hunk 1]
```diff
@@ -300,6 +229,75 @@
          </gmd:CI_ResponsibleParty>
        </gmd:pointOfContact>
        {% endwith %}
+       {% with layer.metadata_author as metadata_author %}
```
Comment (ridoo) [inline geonode/catalogue/templates/catalogue/full_metadata.xml:232]: @ahmdthr what is the relevant change here? Just the ordering?

Comment (ahmdthr) [reply geonode/catalogue/templates/catalogue/full_metadata.xml:232]: Yes, just the placement of this tag. The GDI-DE compliancy requires to only include `pointOfContact` under `<gmd:contact>`, so you can't add `author`, `originator` or any other type of role there.

Comment (ahmdthr) [reply geonode/catalogue/templates/catalogue/full_metadata.xml:232]: It is important to point out that the existing schema validates for ISO-19139, but fails for GDI-DE. And, of course, the new schema is also validated against the ISO-19139 specifications and GDI-DE as well.

**Comment by ridoo** [comment]
@ahmdthr As discussed last week: Move the fixes to `thuenen_atlas` project. Let's decide in the Sprint meeting today to make upstream issue plus PR or not.

**Comment by ridoo** [comment]
Will be merged into `thuenen_atlas`: https://github.com/Thuenen-GeoNode-Development/Sprints/issues/6

**Pull request closed without merging**